### PR TITLE
Add clarification of audio format details

### LINKED ISSUE #33: [SDL 0139] Clarification of audio format details of AudioPassThru
Accepted Evolution Proposal: [Clarification of audio format details of AudioPassThru](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0139-apt-clarification.md)

> The aim of this proposal is to clarify the details of audio data format of AudioPassThru recording.

The 'PerformAudioPassThru' Best Practices Guide should be updated to reflect the `Modification to documents` section of the proposal.

Evolution Proposal Review: https://github.com/smartdevicelink/sdl_evolution/issues/394

### PR DESCRIPTION
**Pull request by theresalech**
Resolves #33.  I also made a few formatting and grammatical changes to the `PerformAudioPassThru` section.

### CODE REVIEW

File: docs/PerformAudioPassThru/index.md [hunk 1]
```diff
@@ -1,16 +1,20 @@
...
-microphone. The audio data could be used in cloud-based and on-line voice
-recognition to achieve dynamic and diversified user interaction, like POI search, information query. Or even record the voice when driver is singing.
-The audio data is in form of PCM, and its sampling rate, bit width, timeout can be set in the `PerformAudioPassThru` request per your requirement. The headunit transmits its supported parameters via the `registerAppInterface` response. 16bit/16kHz is a widely supported configuration, which delivers good audio quality.
+`PerformAudioPassThru` feeds you with audio data through the vehicle’s microphone. The audio data could be used in cloud-based and on-line voice recognition to achieve dynamic and diversified user interaction, like POI search, information query, or even record the voice when the driver is singing. The audio data is in form of PCM, and its sampling rate, bit width, timeout can be set in the `PerformAudioPassThru` request per your requirement. The headunit transmits its supported parameters via the `registerAppInterface` response. 16bit/16kHz is a widely supported configuration, which delivers good audio quality.
```
Comment (joeljfischer) [inline docs/PerformAudioPassThru/index.md]: > `PerformAudioPassThru` feeds you with audio data through the vehicle’s microphone.

remove "with", change "through" to "from"

> The audio data could be used in cloud-based and on-line voice recognition to achieve dynamic and diversified user interaction, like POI search, information query, or even record the voice when the driver is singing.

Change "could" to "can". Remove "and diversified". Change "interaction, like" to "interaction, such as". Remove "the voice".

> The audio data is in form of PCM, and its sampling rate, bit width, timeout can be set in the `PerformAudioPassThru` request per your requirement. The headunit transmits its supported parameters via the `registerAppInterface` response. 16bit/16kHz is a widely supported configuration, which delivers good audio quality.

"The Audio data will be in PCM format. The sample rate, bit width, and timeout can be set, however, the supported parameters will be sent in the `registerAppInterface` response. Generally, 16 bit width, 16KHz sample rate will be supported.

File: docs/PerformAudioPassThru/index.md [hunk 2]
```diff
@@ -1,16 +1,20 @@
...
+The parameter `muteAudio` is used to define whether or not to mute current audio source during AudioPassThru session.
 
 When the `PerformAudioPassThru` is used for voice recognition, `muteAudio` should be set to true to minimize audio interference.
 
 If you want to mix the input audio from `PerformAudioPassThru` session with current audio source, eg. a karaoke app recording both the user's voice and the background music, you can set `muteAudio` to false.
 
 `onAudioPassThru` keeps you updated with the audio data transfer every 250ms.
 
-`EndAudioPassThru` enables you end the audio capture, prematurely. This is useful if your App analyzes the audio level and detects that the user stopped speaking.
+`EndAudioPassThru` enables you to end the audio capture, prematurely. This is useful if your app analyzes the audio level and detects that the user stopped speaking.
```
Comment (joeljfischer) [inline docs/PerformAudioPassThru/index.md]: > `EndAudioPassThru` enables you to end the audio capture, prematurely.

Remove comma.

> This is useful if your app analyzes the audio level and detects that the user stopped speaking.

"has stopped"

File: docs/PerformAudioPassThru/index.md [hunk 3]
```diff
@@ -1,16 +1,20 @@
...
+
+The format of audio data is described as follows:
```
Comment (joeljfischer) [inline docs/PerformAudioPassThru/index.md]: "Additional notes about the audio data format:"

File: docs/PerformAudioPassThru/index.md [hunk 4]
```diff
@@ -1,16 +1,20 @@
...
+
+- It does not include any header like RIFF header at the beginning.
```
Comment (joeljfischer) [inline docs/PerformAudioPassThru/index.md]: There is no header (such as a RIFF header).

File: docs/PerformAudioPassThru/index.md [hunk 5]
```diff
@@ -1,16 +1,20 @@
...
+- The audio sample is in linear PCM format.
+- The audio data includes only one channel (i.e. monaural).
+- For bit rates of 8 bits, the audio samples are unsigned. For bit rates of 16 bits, the audio samples are signed and are in little endian.
```
Comment (joeljfischer) [inline docs/PerformAudioPassThru/index.md]: "For an 8 bit bitrate, the audio data is unsigned. For a 16 bit bitrate, they will be signed and little endian"

**Comment by theresalech** [comment]
@joeljfischer updated per your feedback!

**Comment by theresalech** [comment]
@joeljfischer updated to reflect some of the changes from PR #28.  This should now be ready for your review.  Thanks!

File: docs/PerformAudioPassThru/index.md [hunk 6]
```diff
@@ -1,13 +1,20 @@
...
+The `PerformAudioPassThru` RPC feeds you audio data from the vehicle’s microphone. The audio data can be used in cloud-based and on-line voice recognition to achieve dynamic user interaction, such as POI (point of interest) search, information query, or even record when the driver is singing. The audio data will be in uncompressed PCM format. The sampling rate, bit width, and timeout can be set, however, the supporting parameters will be sent in the `registerAppInterface` response. Generally, 16 bit width, 16kHz sample rate will be supported.
```
Comment (joeljfischer) [inline docs/PerformAudioPassThru/index.md]: > The sampling rate, bit width, and timeout can be set, however, the supporting parameters will be sent in the `registerAppInterface` response.

"supported"

File: docs/PerformAudioPassThru/index.md [hunk 7]
```diff
@@ -1,13 +1,20 @@
...
 
-`EndAudioPassThru` enables you end the audio capture, prematurely. This is useful if your App analyzes the audio level and detects that the user stopped speaking.
+`EndAudioPassThru` enables you to end the audio capture prematurely. This is useful if your app analyzes the audio level and detects that the user has stopped speaking.
+
+Additional notes about the audio data format:
+
+- There is no header (such as a RIFF header).
+- The audio sample is in linear PCM format.
+- The audio data includes only one channel (i.e. monaural).
+- For an 8 bitrate, the audio data is unsigned. For a 16 bitrate, they will be signed and little endian.
```
Comment (joeljfischer) [inline docs/PerformAudioPassThru/index.md]: "8 bitrate" -> "8 bit sample rate", same for 16. Change "they" to "it", referring to "data."

**Pull request merged**